lib/engines/hcktest/playlist.rb

```diff
@@ -60,7 +77,14 @@ def sort_by_duration
     end
 
     def custom_playlist(log)
-      playlist = @project.engine.target['playlist']
+      custom_playlist_file = @project.options.test.playlist
+
+      playlist = if custom_playlist_file.nil?
+                   @project.engine.target['playlist']
+                 else
+                   File.readlines(custom_playlist_file).map(&:chomp)
```

Comment:
`File.readlines` has `chomp` option so use it.

Reply:
fixed